rclcpp_components/src/component_container_mt.cpp

```diff
@@ -22,8 +22,22 @@ int main(int argc, char * argv[])
 {
   /// Component container with a multi-threaded executor.
   rclcpp::init(argc, argv);
+
+  auto options = rclcpp::NodeOptions{}.start_parameter_event_publisher(false);
```

Comment:
why do you need this?
aren't the default NodeOptions that ComponentManager use fine?

Reply:
It's because default node option in `component_manager` is as below.  https://github.com/ros2/rclcpp/blob/86c079de3111631d8ae1a42eb40e1dd2f96c016f/rclcpp_components/include/rclcpp_components/component_manager.hpp#L104-L106
I enable parameter server to set parameters and keep another one as it is. However, I don't know why  this parameter is set to false.

Reply:
you don't need the parameter server enable in this case, you can just pass the initial parameter value through the command line (which doesn't require the parameter server).

Reply:
I changed to use defaut node option at https://github.com/ros2/rclcpp/pull/1708/commits/95bdec1b3c2c4986aba9b320936f06d9f47fe83c.
(The parameter server is enabled to get `thread_num` from it. Expected usage is `ros2 run rclcpp_components component_container_mt --ros-args -p thread_num:=1`.)